### PR TITLE
Dev/comment box update

### PR DESCRIPTION
**Pull request by Joelzac537**
**Introduction:**
Made the comment box and date feature workable

 **Functionality:**
Comment box is showing the date , time and name with space to enter the comment and also date is showing in task after being read from the database

 **How to test:**
By opening Worktrolly and checking the log work comment box by entering appropriate comments

### CODE REVIEW

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
do logWorkComment only remove commentDateTime from everywhere.
Add logWorkComment in index.js instead of commentDateTime
Add logWorkComment in class also

File: public/scripts/frames.js [hunk 1]
```diff
@@ -191,6 +192,11 @@ function fillDataIntoLogWorkPage(id, title, estimatedTime, logWorkTotalTime, wor
     document.getElementById("logWorkTotalTime").innerHTML = logWorkTotalTime;
     document.getElementById("logWorkDone").value = workDone;
     document.getElementById("logWorkRT").innerHTML = estimatedTime - logWorkTotalTime;
+    var today = new Date();
+    var date = today.getDate() + "/" + (today.getMonth() + 1) + "/" + today.getFullYear();
+    var time = today.getHours() + ":" + today.getMinutes();
+    var commentDateTime = date + "|" + time + "|";
+    document.getElementById("logWorkComment").value = commentDateTime + assigneeTaskDescription.innerHTML + " : ";
```
Comment (shivamgoyal7) [inline public/scripts/frames.js:199]: take the date from card description dont write again

**Review by shivamgoyal7 — CHANGES_REQUESTED** [review]
do this also

File: public/scripts/frames.js (hunk 1 above)
Comment (Joelzac537) [reply public/scripts/frames.js]: No change needed, since date in log work comment box should have the date when the work is being logged

**Review by shivamgoyal7 — APPROVED** [review]
Able to merge

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**